css-values-4/Overview.bs

```diff
@@ -3333,7 +3333,7 @@ Computed Value</h3>
 	math expressions mixing both percentages and lengths for widths and heights on
```

Comment:
```suggestion
	math expressions mixing both percentages and non-zero lengths for widths and heights on
```
According to #3482, it is safe to drop 0 length from <length-percentage>. Then how about making it explicit here?

Reply:
> According to #3482, it is safe to drop 0 length from <length-percentage>. Then how about making it explicit here?

SGTM. As far as I can tell, the resolution from https://github.com/w3c/csswg-drafts/issues/3482 isn't in any spec. Are you proposing to add it here explicitly?

Reply:
@tabatkins Could you help?

I'm not familiar with the editing process. What should we do here?